src/server/scripts/Kalimdor/zone_orgrimmar.cpp

```diff
@@ -263,8 +264,563 @@ class npc_thrall_warchief : public CreatureScript
 
 };
 
+/* --------- Herald of War ------------- */
+enum citizenEntries
+{
+    NPC_GRYSHKA         = 31433,
+    NPC_OLVIA           = 31425,
+    NPC_SANA            = 31429,
+    NPC_FELIKA          = 31427,
+    NPC_THUNG           = 31430,
+    NPC_KAJA            = 31423
+};
+
+enum sceneEvents
+{
+    EVENT_SCENE_1       = 1,
+    EVENT_SCENE_2       = 2,
+    EVENT_SCENE_3       = 3,
+    EVENT_SCENE_4       = 4,
+    EVENT_SCENE_5       = 5,
+    EVENT_SCENE_6       = 6,
+    EVENT_SCENE_7       = 7,
+    EVENT_SCENE_8       = 8,
+    EVENT_SCENE_9       = 9,
+    EVENT_SCENE_10      = 10,
+    EVENT_SCENE_11      = 11,
+    EVENT_SCENE_12      = 12,
+    EVENT_SCENE_13      = 13,
+    EVENT_RESET         = 14
+};
+
+enum citizenTalk
+{
+    SAY_GRYSHKA_1       = 0, // When can we reopen our shops? I'm losing gold here!
+    SAY_GRYSHKA_2       = 1, // "This is an outrage!"
+    SAY_OLVIA_1         = 0, // Where is the Warchief?
+    SAY_OLVIA_2         = 1, // What are all these Forsaken doing here?
+    SAY_FELIKA_1        = 0, // What is going on?
+    SAY_FELIKA_2        = 1, // This is an outrage!
+    SAY_THUNG           = 0, // What is going on?
+    SAY_KAJA            = 0, // Why is Thrall allowing this to happen to our city?
+    SAY_SANA            = 0, // We demand answers!
+    SAY_RUNTHAK_1       = 0, // SILENCE!
+    SAY_RUNTHAK_2       = 1, // We are on the brink of all out war with the Alliance!
+    SAY_RUNTHAK_3       = 2, // Tragic events have unfolded in Northrend. The Warchief is doing all that he can to keep us safe.
+    SAY_RUNTHAK_4       = 3  // All services and shops are to remain closed until further notice! That is all!
+};
+
+class npc_overlord_runthak_orgrimmar : public CreatureScript
+{
+public:
+    npc_overlord_runthak_orgrimmar() : CreatureScript("npc_overlord_runthak_orgrimmar") { }
+
+    struct npc_overlord_runthak_orgrimmarAI : public ScriptedAI
+    {
+        npc_overlord_runthak_orgrimmarAI(Creature* creature) : ScriptedAI(creature) 
+        {
+            Initialize();
+            citizenList.clear();
```

Comment:
this line is not needed because a class member std::list<T> is initialized empty by default